From 849a477b3db5b94975b88880138efe80dbe47efd Mon Sep 17 00:00:00 2001
From: Fakeman <55951275+fakemanoan@users.noreply.github.com>
Date: Sun, 31 Jul 2022 22:17:20 +0100
Subject: [PATCH] Allow hardware errors when building

fixes build
---
 cc/config/global.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cc/config/global.go b/cc/config/global.go
index ae731b203..c4f3172e0 100644
--- a/cc/config/global.go
+++ b/cc/config/global.go
@@ -152,6 +152,7 @@ var (
 	WarningAllowedProjects = []string{
 		"device/",
 		"vendor/",
+		"hardware/",
 	}
 
 	// Directories with warnings from Android.mk files.
-- 
2.37.1

